Cargo.toml

```diff
@@ -45,3 +45,7 @@ overflow-checks = true
 panic = 'unwind'
 incremental = true         # This is true because target is cached
 codegen-units = 256
+
+
+[features]
+avx512 = []
```

Comment:
shouldn't this rather be a feature of the `backend` and `powdr` crates instead of the whole repo?

Reply:
Yes, you are right. But I am wondering if we can support the avx512 here, cause powdr uses the stable channel, but the feature `stdarch_x86_avx512` is not available there.